l10n_ch_scan_bvr/wizard/scan_bvr.py

```diff
@@ -345,21 +345,52 @@ def validate_bvr_string(self):
         #
         data = {}
         data['bvr_struct'] = self._get_bvr_structurated(self.bvr_string)
+        partner_bank_model = self.env['res.partner.bank']
+        partner_bank = partner_bank_model
```

Comment:
Why setting an empty record set to partner_bank ? Setting it to false isn't enough ?